src/elements/content-sidebar/__tests__/SidebarPanels.test.js

```diff
@@ -67,6 +67,7 @@ describe('elements/content-sidebar/SidebarPanels', () => {
             ${'/details/versions'}               | ${'VersionsSidebar'}
             ${'/details/versions/1234'}          | ${'VersionsSidebar'}
             ${'/metadata'}                       | ${'MetadataSidebar'}
+            ${'/metadata/filteredTemplates/123'} | ${'MetadataSidebar'}
```

Comment:
from what it looks like above this is a comma delimited string, can we update the mock data to show that?

Reply:
updated 👍